src/workers/aws.ts

```diff
@@ -0,0 +1,39 @@
+import AWS from 'aws-sdk';
+import { WorkerParameters, AWSHandlerResponse } from '../types';
+
+export const spawn = async ({
+  providerSponsorSubscriptions,
+  baseLogOptions,
+  stage,
+}: WorkerParameters): Promise<string> =>
+  // lambda.invoke is synchronous so we need to wrap this in a promise
+  new Promise((resolve, reject) => {
+    // Uses the current region by default
+    const lambda = new AWS.Lambda({ endpoint: 'http://localhost:3002' });
```

Comment:
Why the localhost endpoint?

Reply:
I forgot to remove this (I was testing with serverless-offline).